application/src/test/java/uk/nhs/hee/tis/revalidation/changelog/SetDicsonnectedDoctorsUnderNoticeToNullChangeLogTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.nhs.hee.tis.revalidation.changelog;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.nhs.hee.tis.revalidation.entity.UnderNotice.YES;
+
+import com.github.javafaker.Faker;
+import java.time.LocalDate;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.hee.tis.revalidation.entity.DoctorsForDB;
+import uk.nhs.hee.tis.revalidation.entity.RecommendationStatus;
+import uk.nhs.hee.tis.revalidation.repository.DoctorsForDBRepository;
+
+@ExtendWith(MockitoExtension.class)
+class SetDicsonnectedDoctorsUnderNoticeToNullChangeLogTest {
+
+    SetDisconnectedDoctorsUnderNoticeToNullChangeLog changeLog;
+
+    @Mock
+    DoctorsForDBRepository doctorsForDBRepository;
+
+    @Captor
+    ArgumentCaptor<DoctorsForDB> doctorCaptor;
+
+    List<DoctorsForDB> doctors;
+
+    DoctorsForDB doctor1;
+
+    private final Faker faker = new Faker();
+
+    @BeforeEach
+    public void setup() {
+        changeLog = new SetDisconnectedDoctorsUnderNoticeToNullChangeLog();
+        setupTestData();
+    }
+
+    @Test
+    void shouldSetCorrectTisStatusForEachDoctor() {
+
+        assert (doctor1.getDoctorStatus()).equals(RecommendationStatus.NOT_STARTED);
```

Comment:
1. Using an `assert` here could well trip people up.  I'm going to hazard a guess that you might have wanted to use an `assertEquals()` here.
2. If the purpose of this is to check that all of the test data has been set up correctly; it would be "safer" to use the `List doctors` which is used in this test but I'd check this is something you want to do.
3. I can't see how `DoctorsForDB.doctorStatus` relates to the test or implementation.

Reply:
The answer to all of this is copy + paste = bad 🙈